sympy/series/limitseq.py

```diff
@@ -112,6 +113,7 @@ def _limit_seq(expr, n, trials):
 
     for i in range(trials):
         if not expr.has(Sum):
+            expr = simplify(expr)
```

Comment:
Agreed @normalhuman @asmeurer . Infact, the `test_limit_seq()` test took `30` seconds.

I am now looking to implement the logic only for trigonometric expressions, since they are the ones which can be simplified easily, and are used very commonly. I intend to use `trigsimp` for this.

The following is what I tend to implement --

```
--- a/sympy/series/limitseq.py
+++ b/sympy/series/limitseq.py
@@ -8,8 +8,9 @@
 from sympy.core.power import Pow
 from sympy.core.symbol import Dummy
 from sympy.core.function import PoleError
+from sympy.functions.elementary.trigonometric import TrigonometricFunction
 from sympy.series.limits import Limit
-from sympy.simplify.simplify import simplify
+from sympy.simplify.trigsimp import trigsimp
 
 
 def difference_delta(expr, n=None, step=1):
@@ -111,9 +112,12 @@ def _limit_inf(expr, n):
 def _limit_seq(expr, n, trials):
     from sympy.concrete.summations import Sum
 
+    _trig = TrigonometricFunction
+
     for i in range(trials):
         if not expr.has(Sum):
-            expr = simplify(expr)
+            if expr.has(_trig):
+                expr = trigsimp(expr)
             result = _limit_inf(expr, n)
             if result is not None:
                 return result
``` 

What are your views on it?